### DISCUSSION TITLE
SASL Authentication for Zookeeper and Brokers

### DISCUSSION THREAD

**Discussion by cassanellicarlo** [provectus/kafka-ui#1753] [Q&A] [answered]
Hello,
How can i configure Kafka UI with SASL Authentication for both Broker and Zookeeper?

Currently i have SASL Authentication (GSSAPI) for the Brokers.

    --env KAFKA_CLUSTERS_0_NAME='DC1' \
    --env KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=$BOOTSTRAP_SERVERS_DC1 \
    --env KAFKA_CLUSTERS_0_PROPERTIES_SECURITY_PROTOCOL='SASL_SSL' \
    --env KAFKA_CLUSTERS_0_PROPERTIES_SASL_MECHANISM='GSSAPI' \
    --env KAFKA_CLUSTERS_0_PROPERTIES_SASL_KERBEROS_SERVICE_NAME='kafka' \
    --env KAFKA_CLUSTERS_0_PROPERTIES_SASL_JAAS_CONFIG='com.sun.security.auth.module.Krb5LoginModule required useTicketCache=true;' \
    --env KAFKA_CLUSTERS_0_PROPERTIES_SSL_TRUSTSTORE_LOCATION='/opt/kafka/config/truststore.jks' \
    --env KAFKA_CLUSTERS_0_PROPERTIES_SSL_TRUSTSTORE_PASSWORD='***' \

I have Zookeeper security enabled and i would like to connect kafka ui to Zookeeper via SASL.

**Comment by Haarolean** [comment, marked as answer]
Hi, we've recently dropped support for zookeeper. It's not required for any of the functionality.

**Reply by cassanellicarlo** [reply]
On which docker image tag?

Thanks

**Reply by Haarolean** [reply]
You can skip setting the variable on any version actually, nothing will happen besides red status icon in UI.
